src/Symfony/Component/Workflow/EventListener/GuardListener.php

```diff
@@ -24,13 +24,13 @@
  */
 class GuardListener
 {
-    private $configuration;
-    private $expressionLanguage;
-    private $tokenStorage;
-    private $authorizationChecker;
-    private $trustResolver;
-    private $roleHierarchy;
-    private $validator;
+    protected $configuration;
```

Comment:
That's a no-go: we strongly favor private properties because it helps maintenance a lot.

Reply:
Thanks for the feedback. So as I understand it, it is preferable to create my own new class completely override the listener.guard services declaration.

Reply:
yes - you can decorate the core listener if it serves your purpose, but only via composition, ie using its public API.

Reply:
Since there is a no-go on changing the GuardListener properties visibility and there is other ways to serve my purpose, it seems this pull request can be closed